Repository/ExtendedRepositories/RolesRepository.cs

```diff
@@ -14,56 +14,67 @@ public interface IRolesRepository : IRepository<Role>
         void AssignRoleToUser(string Role, int UserId);
         bool UserHasRole(string Username, string Role);
         bool UserHasRole(int UserId, string Role);
+        void RemoveRoleFormUser(string Role, int UserId);
     }
     public class RolesRepository : Repository<Role>, IRolesRepository
     {
-        private readonly IUserRepository UserRepository;
-        private readonly IRepository<UserRole> UserRoleRepository;
-        public RolesRepository(ApplicationDbContext db, ILogger<RolesRepository> logger, 
+        private readonly IUserRepository _userRepository;
+        private readonly IRepository<UserRole> _userRoleRepository;
+        public RolesRepository(ApplicationDbContext db, ILogger<RolesRepository> logger,
             IUserRepository UserRepository, IRepository<UserRole> UserRoleRepository) : base(db, logger)
         {
-            this.UserRepository = UserRepository;
-            this.UserRoleRepository = UserRoleRepository;
+            this._userRepository = UserRepository;
+            this._userRoleRepository = UserRoleRepository;
         }
+
         public void AssignRoleToUser(string Role, string Username)

```

Comment:
why -_-

Reply:
please elaborate

Reply:
why did you change the variables of this specific class in my code to have _ .... why?

Reply:
all the classes i work in resharper changes the variable names to follow the naming convention to start with _ and a small letter if the variable is private

Reply:
-_-